requirements-dev.txt

```diff
@@ -16,7 +16,7 @@ pydocstyle
 pytest >=3.2
 pytest-mock
 sphinx_compas_theme >=0.13
-sphinx >=3.4
+sphinx ==7.1.2
```

Comment:
is this still required? higher version of sphinx break the theme?

Reply:
yes `7.2.4` breaks the template, I opened https://github.com/compas-dev/sphinx_compas_theme/issues/20. 
Also Sphinx are seemingly dropping python versions like flies. `7.2.0` already doesn't support python 3.8 anymore..